test/functional/home_controller_test.rb

```diff
@@ -18,14 +18,14 @@ class HomeControllerTest < ActionController::TestCase
   view_test "how government works shows the current prime minister" do
     get :how_government_works
 
-    assert_select ".prime-minister p a", "Firstname Lastname"
+    assert_select ".prime-minister > p:nth-child(5) [href]", "Firstname Lastname"
```

Comment:
Not worth changing for this PR but a thought for future: this is not a particularly robust test. If we start adding extra children or removing some of the current content then that target of 5th child becomes invalid and the test needs an update. Probably better to tie things together more with classes generally, I'd have gone for BEM classes across the prime minister metadata myself. At that point you can both style and test the content more directly.

Reply:
Thanks Max - good point. I'll bear that in mind for future.